src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/Install/CompleteStage.php

```diff
@@ -60,14 +60,19 @@ public function getTemplateParams()
 
     public function getResponse(Request $request)
     {
+        $router = $this->container->get('router');
         if ($this->sendEmailToAdmin($request)) {
             $request->getSession()->getFlashBag()->add('success', $this->__('Congratulations! Zikula has been successfully installed.'));
+            $request->getSession()->getFlashBag()->add('info', $this->__f(
+                'Session are currently configured to use the filesystem. It is recommended that you change this to use the database. Click %here% to configure.',
+                ['%here%' => '<a href="' . $router->generate('zikulasecuritycentermodule_config_config') . '">' . $this->__('Security Center') . '</a>']
+            ));
```

Comment:
Would it be possible to change it to the database automatically during/after the `CompleteStage`?